OracleIdentityGovernance/README.md

```diff
@@ -0,0 +1,128 @@
+
+## Prerequisites for OIG on Docker
+
+The following prerequisites are required for building OIG Docker images:
+
+* A working installation of Docker 18.03 or later
+* Docker-compose 1.25.4 or higher
+* A running Oracle Database 12.2.0.1 or later. This database can be installed anywhere. The database must be a supported version for OIG as outlined in [Oracle Fusion Middleware 12c certifications](https://www.oracle.com/technetwork/middleware/fmw-122140-certmatrix-5763476.xlsx), and must meet the requirements as outlined in [About Database Requirements for an Oracle Fusion Middleware Installation](http://www.oracle.com/pls/topic/lookup?ctx=fmw122140&id=GUID-4D3068C8-6686-490A-9C3C-E6D2A435F20A).  If you require a test database, an Oracle Database Docker image can be used. For instructions on installing and running an Oracle Database Docker image refer to section [Appendix A: Using an Oracle Database Docker Image for OIG](#appendix-a-using-an-oracle-database-docker-image-for-oig) at the end of this README.
+
+### Hardware Requirements
+
+| Hardware  | Size                                              |
+| :-------- | :-------------------------------------------------|
+| RAM       | Min 16GB                                          |
+| Disk Space| Min 50GB (ensure 10G+ available in Docker Home)   |
+
+## Prerequisites for OIG on Kubernetes
+
+Refer to the [Prerequisites](https://oracle.github.io/fmw-kubernetes/oig/prerequisites) in the Oracle Identity Governance Kubernetes documentation.
+
+## Installing the OIG Docker Image
+
+An OIG Docker image can be created and/or made available for deployment in a Docker environment in the following ways:
+
+1. Download a pre-built OIG Docker image from [My Oracle Support](https://support.oracle.com) (Patch 31979475).  This image contains the latest bundle patch and one-off patches for Oracle Identity Governance 12.2.1.4.0.
+
+1. Build your own OIG Docker image using the WebLogic Image Tool. Oracle recommends using the Weblogic Image Tool to build your own OIG 12.2.1.4.0 image along with the latest Bundle Patch and any additional patches that you require. For more information, see [Building an OIG Docker Image using Image Tool](https://github.com/oracle/docker-images/tree/master/OracleIdentityGovernance/imagetool/12.2.1.4.0).
+
+
+## OIG Docker Container Configuration
+ 
+To configure the OIG Containers on Docker only, follow the tutorial [Creating Oracle Identity Governance Docker Containers](https://docs.oracle.com/en/middleware/idm/identity-governance/12.2.1.4/tutorial-oig-docker/)
+
+## OIG Kubernetes Configuration
+
+To configure the OIG Containers with Kubernetes see the [Oracle Identity Governance on Kubernetes](https://oracle.github.io/fmw-kubernetes/oig/) documentation.
+
+## Appendix A: Using an Oracle Database Docker Image for OIG
+
+**Note**: Database docker images are not supported in production and should only be used for testing with Docker only configurations and not Kubernetes.
```

Comment:
@gvenzl isn't Oracle Database standalone now supported in production in a container? It's just RAC that's still dev/test, isn't it?